### DISCUSSION TITLE
Changing 10.8.0.x to 10.8.1.x

### DISCUSSION THREAD

**Discussion by parkcheck** [pivpn/pivpn#1404] [Support]
## In raising this issue, I confirm the following:

`{please fill the checkboxes, e.g: [X]}`

- [x] I have read the [documentation](https://docs.pivpn.io)
- [x] I have read and understood the [contributors guide](https://github.com/pivpn/pivpn/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting can be *is* directly related to the pivpn installer script.

### Have you searched for similar issues and solutions?
    (yes/no / which issues?)
yes

### Has your install failed? 
```
yes/no
no
```

#### What type of hardware are you running PiVPN at?

```
Raspberrypi 4

```
#### Output of `uname -a`
```
  OUTPUT HERE / ANSWER HERE
  DO NOT DELETE THE BACK-TICKS 
  PASTE THE OUTPUT INSIDE THE BACK-TICKS
```

#### Output of `cat /etc/os-release`
```
  OUTPUT HERE / ANSWER HERE
  DO NOT DELETE THE BACK-TICKS 
  PASTE THE OUTPUT INSIDE THE BACK-TICKS
```

### If install failed Please provide the console output of  `curl -L https://install.pivpn.io | bash`
```
  OUTPUT HERE / ANSWER HERE
  DO NOT DELETE THE BACK-TICKS 
  PASTE THE OUTPUT INSIDE THE BACK-TICKS
```

### Issue

I have a problem as i use multiple vpn. Another VPN is using 10.8.0 address range as well. 
So I need to change default 10.8.0.0 to 10.8.1.0
I have not found that option on installation so i wanted to do after. 

I updated in server.conf
 `topology subnet
server 10.8.1.0 255.255.255.0
`
Plus the iptables: according to

`# Completed on Wed Nov  3 18:48:44 2021
# Generated by xtables-save v1.8.2 on Wed Nov  3 18:48:44 2021
*filter
:INPUT ACCEPT [842:102022]
:FORWARD ACCEPT [0:0]
:OUTPUT ACCEPT [441:51651]
-A INPUT -i eth0 -p udp -m udp --dport 195 -m comment --comment openvpn-input-rule -j ACCEPT
-A FORWARD -d 10.8.1.0/24 -i eth0 -o tun0 -m conntrack --ctstate RELATED,ESTABLISHED -m comment --comment openvpn-forward-rule -j ACCEPT
-A FORWARD -s 10.8.1.0/24 -i tun0 -o eth0 -m comment --comment openvpn-forward-rule -j ACCEPT
COMMIT
`

reboot and test shows:

`Wed Nov  3 19:52:28 2021 interactive service msg_channel=564
Wed Nov  3 19:52:28 2021 ROUTE_GATEWAY 192.168.1.1/255.255.255.0 I=12 HWADDR=XX:29:76:ad:d7:XX
Wed Nov  3 19:52:28 2021 open_tun
Wed Nov  3 19:52:28 2021 tap-windows6 device [OpenVPN TAP-Windows6] opened
Wed Nov  3 19:52:28 2021 TAP-Windows Driver Version 9.24 
Wed Nov  3 19:52:28 2021 Set TAP-Windows TUN subnet mode network/local/netmask = 10.8.0.0/10.8.0.2/255.255.255.0 [SUCCEEDED]
Wed Nov  3 19:52:28 2021 Notified TAP-Windows driver to set a DHCP IP/netmask of 10.8.0.2/255.255.255.0 on interface {1114416D-AE7A-4C19-B2E1-971678130XXX} [DHCP-serv: 10.8.0.254, lease-time: 31536000]
Wed Nov  3 19:52:28 2021 Successful ARP Flush on interface [15] {1114416D-AE7A-4C19-B2E1-971678130XXX}
Wed Nov  3 19:52:28 2021 MANAGEMENT: >STATE:1635965548,ASSIGN_IP,,10.8.0.2,,,,
Wed Nov  3 19:52:28 2021 IPv4 MTU set to 1500 on interface 15 using service
Wed Nov  3 19:52:28 2021 Blocking outside dns using service succeeded.
Wed Nov  3 19:52:33 2021 TEST ROUTES: 0/1 succeeded len=0 ret=0 a=0 u/d=up
Wed Nov  3 19:52:33 2021 Route: Waiting for TUN/TAP interface to come up...
Wed Nov  3 19:52:38 2021 TEST ROUTES: 0/1 succeeded len=0 ret=0 a=0 u/d=up
Wed Nov  3 19:52:38 2021 Route: Waiting for TUN/TAP interface to come up...
`
i dont get why it still goes to 10.8.0.X

The opevpn DHCP seems to be on 10.8.0.254 -> so assume that i must also be able to change to 10.8.1.somewhat

Strange is once i reboot the pi, the iptables are back to 10.8.0.1 even though i updated 
/etc/network/interfaces  with "pre-up iptables-restore"

can I somehow reinstall with 10.8.1.X settings or any hint where i need to change also in the current installation?

### Console output of      `curl -L install.pivpn.io | bash`
```
  OUTPUT HERE
  DO NOT DELETE THE BACK-TICKS 
  PASTE THE OUTPUT INSIDE THE BACK-TICKS
```

### Console output of      `pivpn add` or `pivpn add nopass`
```
  OUTPUT HERE
  DO NOT DELETE THE BACK-TICKS 
  PASTE THE OUTPUT INSIDE THE BACK-TICKS
```

### Console output of      `pivpn debug`

```
  OUTPUT HERE
  DO NOT DELETE THE BACK-TICKS 
  PASTE THE OUTPUT INSIDE THE BACK-TICKS
```
### Have you taken any steps towards solving your issue?
```
  which?
```

**Comment by coolapso** [comment]
Hi, thanks for reaching out, 

> [x] I have read the [documentation](https://docs.pivpn.io)

Please make sure that you have read the documentation, as it seems you clearly did not. 

at: https://github.com/pivpn/pivpn/#general-guidelines AND https://docs.pivpn.io/#general-guidelines

> The Github issues page is not intended for general support usage or feature requests
PiVPN team may close any Discussion/Issue without any warning if they don't follow the Guidelines


Please  make sure you properly fill the template with the requested data! 
Please properly format the template, there is even a preview tab to help you see how it looks like. 


Moving this to discussions.

**Comment by parkcheck** [comment]
I have had a look at the docs but could not find anything related to my issue.

**Comment by TinCanTech** [comment]
The client IP address is set in the client config directory `/etc/openvpn/ccd`
